<ui version="4.0" >
 <class>GuiSmoothSurface</class>
 <widget class="QDialog" name="GuiSmoothSurface" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>smooth surface</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="engrid.qrc" >
    <normaloff>:/icons/resources/icons/G.png</normaloff>:/icons/resources/icons/G.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="doubleSpinBox" >
         <property name="maximum" >
          <double>180.000000000000000</double>
         </property>
         <property name="value" >
          <double>45.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>feature angle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>iterations</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="spinBox" >
         <property name="maximum" >
          <number>1000</number>
         </property>
         <property name="singleStep" >
          <number>10</number>
         </property>
         <property name="value" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="SmoothMethod" >
         <item>
          <property name="text" >
           <string>Method 0</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Method 1</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Method 2</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Method 3</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Method 4</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Method 5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Boundary code</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lineEdit_BoundaryCode" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>boundary codes</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QListWidget" name="listWidget" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pushButtonHelp" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="engrid.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GuiSmoothSurface</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GuiSmoothSurface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
